inc/classes/preload/class-sitemap.php

```diff
@@ -108,6 +108,19 @@ public function process_sitemap( $sitemap_url, $urls = [] ) {
 		if ( is_wp_error( $sitemap ) ) {
 			// Translators: %1$s is a XML sitemap URL, %2$s is the error message, %3$s = opening link tag, %4$s = closing link tag..
 			$errors['errors'][] = sprintf( __( 'Sitemap preload encountered an error. Could not gather links on %1$s because of the following error: %2$s. %3$sLearn more%4$s.', 'rocket' ), $sitemap_url, $sitemap->get_error_message(), '<a href="https://docs.wp-rocket.me/article/1065-sitemap-preload-is-slow-or-some-pages-are-not-preloaded-at-all#failed-preload" rel="noopener noreferrer" target=_"blank">', '</a>' );
+			
+			// Attempt to use the fallback method.
+			$fallback_urls = $this->get_urls();
+			
+			if ( ! empty( $fallback_urls ) ) {
+				
+				$urls = array_merge( $urls, $fallback_urls );
+				
+				$errors['errors'][] = __( 'WP Rocket will use a fallback method to preload your website.', 'rocket' );
```

Comment:
I think we determined we didn't want to communicate the use of the fallback to the users

Reply:
Removed it.